docs/source/protocol.rst

```diff
@@ -30,28 +30,29 @@ Network Send Number
 - Begins at 0 on each client and goes up by 1 each time we attempt a metrics network upload.
 - Does not go up on network retries.
 
-Relative Timestamp
-++++++++++++++++++
+Image
++++++
 
-- 64 bit signed integer
-- GVariant symbol: ``x``
-- See: http://linux.die.net/man/3/clock_gettime
+- String
+- GVariant symbol: ``s``
 
-Absolute Timestamp
-++++++++++++++++++
+Site
+++++
 
-- 64 bit signed integer
-- GVariant symbol: ``x``
-- Nanoseconds since the Unix epoch
-- See: http://linux.die.net/man/3/clock_gettime
+- String
```

Comment:
Dictionary from string to string

Reply:
Could also lift its description from `LocationLabel`

Reply:
> Dictionary from string to string

Copy-pasting is bad 👎🏽.

> Could also lift its description from `LocationLabel`

Sure.

Reply:
Can't see the description!

Reply:
Idem.